pages/add-game/index.js

```diff
@@ -95,7 +95,7 @@ export default function AddGame({ db, myOwnCards, setMyOwnCards }) {
 
   return (
     <MainForm>
-      <form onSubmit={handleSubmit}>
+      <Section className={step === 0 ? "active" : "hidden"}>
```

Comment:
Since we don't style elements via `className` you will have to use `styled-components` here too. If necessary, use props to make styling dynamic.